browser/ts/encoding.ts

```diff
@@ -0,0 +1,291 @@
+import { SignedOrder } from '@0x/order-utils';
```

Comment:
How about renaming this file to __wrapper_conversions.ts__? It seems like it is focused on converting to and from the `Wrapper` types.